src/liballoc/slice.rs

```diff
@@ -206,8 +206,9 @@ impl<T> [T] {
     /// the ordering is not total, the order of the elements is unspecified. An order is a
     /// total order if it is (for all `a`, `b` and `c`):
     ///
-    /// * total and antisymmetric: exactly one of `a < b`, `a == b` or `a > b` is true, and
-    /// * transitive, `a < b` and `b < c` implies `a < c`. The same must hold for both `==` and `>`.
+    /// * totality: `a < b` or `b < a`
+    /// * antisymmetric: `a < b` and `b < a`, then `a == b`
+    /// * transitive: `a < b` and `b < c` implies `a < c`. The same must hold for both `==` and `>`.
```

Comment:
"Totality, antisymmetric, transitive" is *noun*, *adjective*, *adjective*, which is an awkward list. The list would be clearer as *noun*, *noun*, *noun*, or *adjective*, *adjective*, *adjective*.